### ISSUE TITLE
Settings gone from 4chan X menu?

### ISSUE THREAD

**Issue by kk2k** [dnsev/4cs#67]
The media player itself works but the settings seems to be gone from my menu.
![capture](https://cloud.githubusercontent.com/assets/8011667/3420215/04124190-fe9d-11e3-8fb7-bf49c075c8de.PNG)

**Comment by milkytiptoe** [comment]
@dnsev this bug is because 'AddMenuEntry' was nuked from 4chan X. In case you planned on fixing.

**Comment by dnsev** [comment]
Should have a fallback link in [5.1.0.1](https://github.com/dnsev/4cs/commit/c1669be70d76e3d2478721130e97d31d29f231ab)

**Comment by kk2k** [comment]
Where? I read the changelog and it said "at the top of the page" but I can't find it even when I turn 4chan X off.

**Comment by dnsev** [comment]
`[Media Player] [Return] [Catalog] [Bottom]` up at the top is where it appears for me.

What version of 4chan-x are you using?

**Comment by kk2k** [comment]
I don't see it, I'm using 3.20.15
![capture](https://cloud.githubusercontent.com/assets/8011667/3425071/f7150d7a-fff4-11e3-9800-3a2d011d8f49.PNG)

**Comment by dnsev** [comment]
Odd. It [works](https://cloud.githubusercontent.com/assets/3087287/3425167/01199cb4-fffa-11e3-9ece-688cd359dcd1.png) for me. I assume we're talking Firefox here? (Not that I can test in Chrome anyway because 4cs isn't on the webstore or some other bullshit reason.)

Not sure what I can suggest at this point. Maybe try [another fork of 4chan-x](https://github.com/ccd0/4chan-x)?

**Comment by dnsev** [comment]
Wait, this may be because of Greasemonkey 2.0, if you have that. Checking that...

---

Okay if you have GM2.0, 4cs won't work because of all the hack-y things it does to try to run quickly (for the sounds parts.) Basically, the script throws errors during the setup, making the rest of the script never execute.

Likely, other features such as media player events also won't work because of this.

If you really want to continue using 4cs, use GM1.15. Otherwise, abandon it because problems this deep will not be fixed.

**Comment by kk2k** [comment]
Yes to Firefox, using Greasemonkey 1.15. It should be there even with 4chan X off, right? All I know is it's not showing up whether X is on or off but it was working when it was in the dropdown menu.

**Comment by dnsev** [comment]
Okay weird that it's not working in GM1.15. If X is off, it should appear up at the top near `[Media Player] [Settings] [Home]`.

If you know anything about Javascript, you might be able to find out if the script is erroring somewhere in `$(document).ready(function () {` using try/catch's or alerts. If not, I have no idea how to tell what's causing this problem.
